.travis.yml

```diff
@@ -79,9 +81,27 @@ install:
     fi
 
 script:
-  - if [ -n "$BUILD" ]; then cmake . -DMETEREXEC_ROOTACCESS=OFF -DENABLE_OCR=ON -DSML_HOME=/usr/local/src/libsml/sml && make && make test; fi
-  - |
-    if [ -n "$FORMAT" ]; then
-      clang-format -i include/**/*.{h,hpp} src/**/*.cpp tests/**/*.{cpp,hpp}
+  - pwd
+  - ls
+  - echo BUILD=$BUILD
+  - echo FORMAT=$FORMAT
+  - if [ -n "$BUILD" ]; then
+      echo "Executing build... in $installDir";
+      cd $installDir;
+      mkdir -p build;
+      cd build;
+      cmake .. -DMETEREXEC_ROOTACCESS=OFF -DENABLE_OCR=ON -DSML_HOME=/usr/local/src/libsml/sml && make && make test;
+    fi
+  - if [ -n "$FORMAT" ]; then
+      echo "Executing clang-format check... in $installDir";
+      cd $installDir;
+      pwd;
+      clang-format --version;
+      git --version;
+      find include src tests -type f -name '*.h' -o -name '*.cpp' -o -name '*.hpp' | xargs -I{} -P1 clang-format -i -style=file {} ;
+      echo "Git status after clang-format:";
```

Comment:
wofür brauchen wir den git status- das passiert doch unten im test?

Reply:
ja, aber sie helfen beim Debugger. z.B. war der vorherige include/**/... Ausdruck falsch, da er auf fzf/zsh Funktionalität basiert. Die git status Meldung war irgendwas wie "file not found...". Jetzt sieht man solche Fehler.
(die --version könnte man weglassen)